src/response.js

```diff
@@ -0,0 +1,28 @@
+export default (originalResponse = {}) => {
+  const response = {
+    response: originalResponse,
+    body: null,
+    get statusCode () {
+      return this.response.status
+    },
+    get statusMessage () {
+      return this.response.statusText
+    },
+    async _getJson () {
```

Comment:
why the underscore prefix?

Reply:
because one shouldn't use them in "client" scripts (because you can't call them twice for example), but makes life easier in `transport.js`. So they're kinda "private", but not really in the sense of the actual private;) 

Any idea for a better solution?

Reply:
hmm, but you use it only in `src/index.js`, no? And only once.. Maybe just make simple functions without encapsulating them within this response object?

Reply:
Ok, changed that now too.